back/pkg/roomcontroller/models/room.go

```diff
@@ -3,15 +3,9 @@ package models
 import "studi-guide/pkg/navigation"
 
 type Room struct {
-	MapItem
-	Id       int
-	PathNode navigation.PathNode
-}
-
-type ConnectorSpace struct {
 	MapItem
 	Id        int
-	PathNodes []navigation.PathNode
+	PathNodes []*navigation.PathNode
```

Comment:
Sollen wir das nicht von den DB Mappern machen lassen, dass der Raum weiterhin nur eine PathNode hat und wir nicht bei jedem Zugriff manuell auf `PathNodes[0]` zugreifen müssen? So hätten wir auch eine Fehlerquelle im zukünftigen Code weniger, falls durch einen blöden Zufall der array oder der Pointer mal nil ist.

Reply:
Du meinst dass der Raum noch ein extra pathnode Field hat, welches der „MasterNode“ is ?

Reply:
Habe gerade gesehen dass das ja schon durch die DB validiert wird. Dort ist die PathNode schon als Required eingetragen.